pkgs/ipfs/go-ipfs/registry.yaml

```diff
@@ -0,0 +1,9 @@
+packages:
+  - type: github_release
+    repo_owner: ipfs
+    repo_name: go-ipfs
+    asset: 'go-ipfs_{{.Version}}_{{.OS}}-{{.Arch}}.tar.gz'
+    description: IPFS is a global, versioned, peer-to-peer filesystem.
```

Comment:
```suggestion
    description: IPFS is a global, versioned, peer-to-peer filesystem
```

https://github.com/aquaproj/aqua-registry/blob/main/CONTRIBUTING.md#remove-a-period-from-the-end-of-the-description